From bee17ea9d3b97f05a5e64c0340051c8706c19dbb Mon Sep 17 00:00:00 2001
From: Arnon Kaufman <akaufman@habana.ai>
Date: Sat, 27 Apr 2024 12:07:28 +0300
Subject: [PATCH] [SW-15774] board: support full path in its files

Current included files within the its files are searched only in the
working directory, not supporting full path.

Change-Id: I2c4f4782400dea2a209602daef99b62a7465a9a2
Signed-off-by: Arnon Kaufman <akaufman@habana.ai>
---

diff --git a/board/habanalabs/common/agent-fw_loader.its b/board/habanalabs/common/agent-fw_loader.its
index b773dc9..e68ebd3 100644
--- a/board/habanalabs/common/agent-fw_loader.its
+++ b/board/habanalabs/common/agent-fw_loader.its
@@ -16,7 +16,7 @@
 	images {
 		security {
 			description = "Habana security image";
-			data = /incbin/("@DEV_PREFIX@-security-arc.bin.@ENDIANNESS@");
+			data = /incbin/("@FILES_PATH_2@@DEV_PREFIX@-security-arc.bin.@ENDIANNESS@");
 			compression = "none";
 			offset = <@SEC_OFFSET@>;
 			hash {
@@ -26,7 +26,7 @@
 
 		img-ptr {
 			description = "Habana image pointers";
-			data = /incbin/("@DEV_PREFIX@-@IMG_BIN@.bin.@ENDIANNESS@");
+			data = /incbin/("@FILES_PATH_1@@DEV_PREFIX@-@IMG_BIN@.bin.@ENDIANNESS@");
 			compression = "none";
 			offset = <@IMG_PTR_OFFSET@>;
 			hash {
@@ -36,7 +36,7 @@
 
 		ppboot {
 			description = "Habana PPboot";
-			data = /incbin/("@DEV_PREFIX@-@PPBOOT_BIN@.bin.@ENDIANNESS@");
+			data = /incbin/("@FILES_PATH_1@@DEV_PREFIX@-@PPBOOT_BIN@.bin.@ENDIANNESS@");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
@@ -48,7 +48,7 @@
 
 		preboot {
 			description = "Habana Preboot";
-			data = /incbin/("@DEV_PREFIX@-@PREBOOT_BIN@.bin.@ENDIANNESS@");
+			data = /incbin/("@FILES_PATH_1@@DEV_PREFIX@-@PREBOOT_BIN@.bin.@ENDIANNESS@");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
diff --git a/board/habanalabs/common/agent-fw_loader_erom.its b/board/habanalabs/common/agent-fw_loader_erom.its
index d3f6eb4..55bd6b5 100644
--- a/board/habanalabs/common/agent-fw_loader_erom.its
+++ b/board/habanalabs/common/agent-fw_loader_erom.its
@@ -15,7 +15,7 @@
 	images {
 		erom {
 			description = "Erom";
-			data = /incbin/("@HL_DEV@-erom.bin.@ENDIANNESS@");
+			data = /incbin/("@FILES_PATH_2@@HL_DEV@-erom.bin.@ENDIANNESS@");
 			type = "standalone";
 			compression = "none";
 			offset = <0>;
diff --git a/board/habanalabs/common/post-image-exports.sh b/board/habanalabs/common/post-image-exports.sh
index cb33fba..47db09a 100755
--- a/board/habanalabs/common/post-image-exports.sh
+++ b/board/habanalabs/common/post-image-exports.sh
@@ -15,3 +15,5 @@
 ITS_SED_PARAMS+="s/@HL_SEC_FW_VERSION@/${HL_SEC_FW_VERSION}/g;";
 ITS_SED_PARAMS+="s/@SHA_VERSION@/${SHA_VERSION}/g;"
 ITS_SED_PARAMS+="s#@HL_UBOOT_BIN_FILE@#${HL_UBOOT_BIN_FILE}#g;"
+ITS_SED_PARAMS+="s#@FILES_PATH_1@#${BINS_DIR}#g;"
+ITS_SED_PARAMS+="s#@FILES_PATH_2@#${BINS_PROD_DIR}#g;"
diff --git a/board/habanalabs/common/spi-fpga-legacy.its b/board/habanalabs/common/spi-fpga-legacy.its
index 0c1a77c..e1de368 100644
--- a/board/habanalabs/common/spi-fpga-legacy.its
+++ b/board/habanalabs/common/spi-fpga-legacy.its
@@ -15,7 +15,7 @@
 	images {
 		btl-s1 {
 			description = "Habana BTL primary";
-			data = /incbin/("s1.bin.le");
+			data = /incbin/("@FILES_PATH_1@s1.bin.le");
 			compression = "none";
 			offset = <0x00000000>;
 			hash {
@@ -25,7 +25,7 @@
 
 		btl-s2 {
 			description = "Habana BTL secondary";
-			data = /incbin/("s2.bin.le");
+			data = /incbin/("@FILES_PATH_1@s2.bin.le");
 			compression = "none";
 			offset = <0x00010000>;
 			hash {
@@ -35,7 +35,7 @@
 
 		btl-s3 {
 			description = "Habana BTL prstn";
-			data = /incbin/("s3.bin.le");
+			data = /incbin/("@FILES_PATH_1@s3.bin.le");
 			compression = "none";
 			offset = <0x00020000>;
 			hash {
@@ -45,7 +45,7 @@
 
 		btl-s4 {
 			description = "Habana BTL pcie";
-			data = /incbin/("s4.bin.le");
+			data = /incbin/("@FILES_PATH_1@s4.bin.le");
 			compression = "none";
 			offset = <0x00030000>;
 			hash {
@@ -55,7 +55,7 @@
 
 		img-ptr {
 			description = "Habana image pointers";
-			data = /incbin/("@HL_DEV@-images-pointers.bin.le");
+			data = /incbin/("@FILES_PATH_2@@HL_DEV@-images-pointers.bin.le");
 			compression = "none";
 			offset = <0x00060000>;
 			hash {
@@ -65,7 +65,7 @@
 
 		preboot {
 			description = "Habana Preboot";
-			data = /incbin/("@HL_DEV@-preboot.bin.le");
+			data = /incbin/("@FILES_PATH_2@@HL_DEV@-preboot.bin.le");
 			type = "standalone";
 			arch = "arm64";
 			compression = "none";
diff --git a/board/habanalabs/common/uboot-zephyr.its b/board/habanalabs/common/uboot-zephyr.its
index dc26edc..a4f7860 100644
--- a/board/habanalabs/common/uboot-zephyr.its
+++ b/board/habanalabs/common/uboot-zephyr.its
@@ -37,7 +37,7 @@
 
 		zephyr-pid-iccm {
 			description = "Zephyr pid ICCM";
-			data = /incbin/("@HL_DEV@-zephyr-pid-iccm.bin.le");
+			data = /incbin/("@FILES_PATH_1@@HL_DEV@-zephyr-pid-iccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
@@ -50,7 +50,7 @@
 
 		zephyr-pid-dccm {
 			description = "Zephyr pid DCCM";
-			data = /incbin/("@HL_DEV@-zephyr-pid-dccm.bin.le");
+			data = /incbin/("@FILES_PATH_1@@HL_DEV@-zephyr-pid-dccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
@@ -63,7 +63,7 @@
 
 		zephyr-mgmt-iccm {
 			description = "Zephyr mgmt ICCM";
-			data = /incbin/("@HL_DEV@-zephyr-mgmt-iccm.bin.le");
+			data = /incbin/("@FILES_PATH_1@@HL_DEV@-zephyr-mgmt-iccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
@@ -76,7 +76,7 @@
 
 		zephyr-mgmt-dccm {
 			description = "Zephyr mgmt DCCM";
-			data = /incbin/("@HL_DEV@-zephyr-mgmt-dccm.bin.le");
+			data = /incbin/("@FILES_PATH_1@@HL_DEV@-zephyr-mgmt-dccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
diff --git a/board/habanalabs/common/uboot-zephyr_reloc.its b/board/habanalabs/common/uboot-zephyr_reloc.its
index e823a4d..71ee1f8 100644
--- a/board/habanalabs/common/uboot-zephyr_reloc.its
+++ b/board/habanalabs/common/uboot-zephyr_reloc.its
@@ -48,7 +48,7 @@
 
 		zephyr-pid-iccm {
 			description = "Zephyr pid ICCM";
-			data = /incbin/("@HL_DEV@-zephyr-pid-iccm.bin.le");
+			data = /incbin/("@FILES_PATH_1@@HL_DEV@-zephyr-pid-iccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
@@ -61,7 +61,7 @@
 
 		zephyr-pid-dccm {
 			description = "Zephyr pid DCCM";
-			data = /incbin/("@HL_DEV@-zephyr-pid-dccm.bin.le");
+			data = /incbin/("@FILES_PATH_1@@HL_DEV@-zephyr-pid-dccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
@@ -74,7 +74,7 @@
 
 		zephyr-mgmt-iccm {
 			description = "Zephyr mgmt ICCM";
-			data = /incbin/("@HL_DEV@-zephyr-mgmt-@ARC_MGMT_ICCM_EXT@");
+			data = /incbin/("@FILES_PATH_1@@HL_DEV@-zephyr-mgmt-@ARC_MGMT_ICCM_EXT@");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
@@ -87,7 +87,7 @@
 
 		zephyr-mgmt-dccm {
 			description = "Zephyr mgmt DCCM";
-			data = /incbin/("@HL_DEV@-zephyr-mgmt-@ARC_MGMT_DCCM_EXT@");
+			data = /incbin/("@FILES_PATH_1@@HL_DEV@-zephyr-mgmt-@ARC_MGMT_DCCM_EXT@");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
@@ -100,7 +100,7 @@
 
 		zephyr-mgmt-sram {
 			description = "Zephyr mgmt SRAM";
-			data = /incbin/("@HL_DEV@-zephyr-mgmt-@ARC_MGMT_SRAM_EXT@");
+			data = /incbin/("@FILES_PATH_1@@HL_DEV@-zephyr-mgmt-@ARC_MGMT_SRAM_EXT@");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
@@ -113,7 +113,7 @@
 
 		zephyr-mgmt-reloc {
 			description = "Zephyr mgmt RELOC";
-			data = /incbin/("@HL_DEV@-zephyr-mgmt-reloc.bin.le");
+			data = /incbin/("@FILES_PATH_1@@HL_DEV@-zephyr-mgmt-reloc.bin.le");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
diff --git a/board/habanalabs/common/uboot-zephyr_sram.its b/board/habanalabs/common/uboot-zephyr_sram.its
index 94ed2a7..0a1a4aa 100644
--- a/board/habanalabs/common/uboot-zephyr_sram.its
+++ b/board/habanalabs/common/uboot-zephyr_sram.its
@@ -45,7 +45,7 @@
 
 		zephyr-pid-iccm {
 			description = "Zephyr pid ICCM";
-			data = /incbin/("@HL_DEV@-zephyr-pid-iccm.bin.le");
+			data = /incbin/("@FILES_PATH_1@@HL_DEV@-zephyr-pid-iccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
@@ -58,7 +58,7 @@
 
 		zephyr-pid-dccm {
 			description = "Zephyr pid DCCM";
-			data = /incbin/("@HL_DEV@-zephyr-pid-dccm.bin.le");
+			data = /incbin/("@FILES_PATH_1@@HL_DEV@-zephyr-pid-dccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
@@ -71,7 +71,7 @@
 
 		zephyr-mgmt-iccm {
 			description = "Zephyr mgmt ICCM";
-			data = /incbin/("@HL_DEV@-zephyr-mgmt-@ARC_MGMT_ICCM_EXT@");
+			data = /incbin/("@FILES_PATH_1@@HL_DEV@-zephyr-mgmt-@ARC_MGMT_ICCM_EXT@");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
@@ -84,7 +84,7 @@
 
 		zephyr-mgmt-dccm {
 			description = "Zephyr mgmt DCCM";
-			data = /incbin/("@HL_DEV@-zephyr-mgmt-@ARC_MGMT_DCCM_EXT@");
+			data = /incbin/("@FILES_PATH_1@@HL_DEV@-zephyr-mgmt-@ARC_MGMT_DCCM_EXT@");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
@@ -97,7 +97,7 @@
 
 		zephyr-mgmt-sram {
 			description = "Zephyr mgmt SRAM";
-			data = /incbin/("@HL_DEV@-zephyr-mgmt-@ARC_MGMT_SRAM_EXT@");
+			data = /incbin/("@FILES_PATH_1@@HL_DEV@-zephyr-mgmt-@ARC_MGMT_SRAM_EXT@");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
diff --git a/board/habanalabs/common/uboot_pldm_regression.its b/board/habanalabs/common/uboot_pldm_regression.its
index 1ea1e04..ecdc872 100644
--- a/board/habanalabs/common/uboot_pldm_regression.its
+++ b/board/habanalabs/common/uboot_pldm_regression.its
@@ -9,7 +9,7 @@
 	images {
 		zephyr-pid-iccm {
 			description = "Zephyr pid ICCM";
-			data = /incbin/("@HL_DEV@-zephyr-pid-iccm.bin.le");
+			data = /incbin/("@FILES_PATH_1@@HL_DEV@-zephyr-pid-iccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
@@ -22,7 +22,7 @@
 
 		zephyr-pid-dccm {
 			description = "Zephyr pid DCCM";
-			data = /incbin/("@HL_DEV@-zephyr-pid-dccm.bin.le");
+			data = /incbin/("@FILES_PATH_1@@HL_DEV@-zephyr-pid-dccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
@@ -35,7 +35,7 @@
 
 		zephyr-mgmt-iccm {
 			description = "Zephyr mgmt ICCM";
-			data = /incbin/("@HL_DEV@-zephyr-mgmt-iccm.bin.le");
+			data = /incbin/("@FILES_PATH_1@@HL_DEV@-zephyr-mgmt-iccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
@@ -48,7 +48,7 @@
 
 		zephyr-mgmt-dccm {
 			description = "Zephyr mgmt DCCM";
-			data = /incbin/("@HL_DEV@-zephyr-mgmt-dccm.bin.le");
+			data = /incbin/("@FILES_PATH_1@@HL_DEV@-zephyr-mgmt-dccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
diff --git a/board/habanalabs/common/zephyr.its b/board/habanalabs/common/zephyr.its
index 40fde25..0a7ad5e 100644
--- a/board/habanalabs/common/zephyr.its
+++ b/board/habanalabs/common/zephyr.its
@@ -9,7 +9,7 @@
 	images {
 		zephyr-pid-iccm {
 			description = "Zephyr pid ICCM";
-			data = /incbin/("@DEV_NAME@-zephyr-pid-iccm.bin.le");
+			data = /incbin/("@FILES_PATH_2@@DEV_NAME@-zephyr-pid-iccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_D0_PSOC_ARC0@>;
@@ -23,7 +23,7 @@
 
 		zephyr-pid-dccm {
 			description = "Zephyr pid DCCM";
-			data = /incbin/("@DEV_NAME@-zephyr-pid-dccm.bin.le");
+			data = /incbin/("@FILES_PATH_2@@DEV_NAME@-zephyr-pid-dccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_D0_PSOC_ARC0@>;
@@ -36,7 +36,7 @@
 
 		zephyr-mgmt-iccm {
 			description = "Zephyr mgmt ICCM";
-			data = /incbin/("@DEV_NAME@-zephyr-mgmt-iccm.bin.le");
+			data = /incbin/("@FILES_PATH_2@@DEV_NAME@-zephyr-mgmt-iccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_D0_PSOC_ARC1@>;
@@ -50,7 +50,7 @@
 
 		zephyr-mgmt-dccm {
 			description = "Zephyr mgmt DCCM";
-			data = /incbin/("@DEV_NAME@-zephyr-mgmt-dccm.bin.le");
+			data = /incbin/("@FILES_PATH_2@@DEV_NAME@-zephyr-mgmt-dccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_D0_PSOC_ARC1@>;
@@ -63,7 +63,7 @@
 
 		zephyr-event-iccm {
 			description = "Zephyr event ICCM";
-			data = /incbin/("@DEV_NAME@-zephyr-event-iccm.bin.le");
+			data = /incbin/("@FILES_PATH_2@@DEV_NAME@-zephyr-event-iccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_D0_PSOC_ARC2@>;
@@ -77,7 +77,7 @@
 
 		zephyr-event-dccm {
 			description = "Zephyr event DCCM";
-			data = /incbin/("@DEV_NAME@-zephyr-event-dccm.bin.le");
+			data = /incbin/("@FILES_PATH_2@@DEV_NAME@-zephyr-event-dccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_D0_PSOC_ARC2@>;
@@ -90,7 +90,7 @@
 
 		external_fw {
 			description = "NIC PHY FW image";
-			data = /incbin/("@DEV_NAME@-eth-phy.bin.le");
+			data = /incbin/("@FILES_PATH_2@@DEV_NAME@-eth-phy.bin.le");
 			compression = "none";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_D0_PSOC_ARC1@>;
diff --git a/board/habanalabs/fs1_arc/fs1-zephyr.its b/board/habanalabs/fs1_arc/fs1-zephyr.its
index d694e41..ed8daf5 100644
--- a/board/habanalabs/fs1_arc/fs1-zephyr.its
+++ b/board/habanalabs/fs1_arc/fs1-zephyr.its
@@ -9,7 +9,7 @@
 	images {
 		zephyr-pid-iccm {
 			description = "Zephyr pid ICCM";
-			data = /incbin/("@DEV_NAME@-zephyr-pid-iccm.bin.le");
+			data = /incbin/("@FILES_PATH_2@@DEV_NAME@-zephyr-pid-iccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_D0_PSOC_ARC0@>;
@@ -23,7 +23,7 @@
 
 		zephyr-pid-dccm {
 			description = "Zephyr pid DCCM";
-			data = /incbin/("@DEV_NAME@-zephyr-pid-dccm.bin.le");
+			data = /incbin/("@FILES_PATH_2@@DEV_NAME@-zephyr-pid-dccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_D0_PSOC_ARC0@>;
@@ -36,7 +36,7 @@
 
 		zephyr-mgmt-iccm {
 			description = "Zephyr mgmt ICCM";
-			data = /incbin/("@DEV_NAME@-zephyr-mgmt-iccm.bin.le");
+			data = /incbin/("@FILES_PATH_2@@DEV_NAME@-zephyr-mgmt-iccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_D0_PSOC_ARC1@>;
@@ -50,7 +50,7 @@
 
 		zephyr-mgmt-dccm {
 			description = "Zephyr mgmt DCCM";
-			data = /incbin/("@DEV_NAME@-zephyr-mgmt-dccm.bin.le");
+			data = /incbin/("@FILES_PATH_2@@DEV_NAME@-zephyr-mgmt-dccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_D0_PSOC_ARC1@>;
diff --git a/board/habanalabs/gaudi2/agent-fw_loader_erase-primary-preboot.its b/board/habanalabs/gaudi2/agent-fw_loader_erase-primary-preboot.its
index a61bef5..ca2d3d7 100644
--- a/board/habanalabs/gaudi2/agent-fw_loader_erase-primary-preboot.its
+++ b/board/habanalabs/gaudi2/agent-fw_loader_erase-primary-preboot.its
@@ -16,7 +16,7 @@
 	images {
 		preboot {
 			description = "Habana Preboot";
-			data = /incbin/("empty.bin");
+			data = /incbin/("@FILES_PATH_1@empty.bin");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
diff --git a/board/habanalabs/gaudi2/agent-fw_loader_legacy.its b/board/habanalabs/gaudi2/agent-fw_loader_legacy.its
index 79d19e4..df6bba8 100644
--- a/board/habanalabs/gaudi2/agent-fw_loader_legacy.its
+++ b/board/habanalabs/gaudi2/agent-fw_loader_legacy.its
@@ -16,7 +16,7 @@
 	images {
 		btl-s1 {
 			description = "Habana BTL primary";
-			data = /incbin/("s1.bin.le");
+			data = /incbin/("@FILES_PATH_1@s1.bin.le");
 			compression = "none";
 			offset = <0x00000000>;
 			hash {
@@ -26,7 +26,7 @@
 
 		btl-s2 {
 			description = "Habana BTL secondary";
-			data = /incbin/("s2.bin.le");
+			data = /incbin/("@FILES_PATH_1@s2.bin.le");
 			compression = "none";
 			offset = <0x00010000>;
 			hash {
@@ -36,7 +36,7 @@
 
 		btl-s3 {
 			description = "Habana BTL prstn 1";
-			data = /incbin/("s3.bin.le");
+			data = /incbin/("@FILES_PATH_1@s3.bin.le");
 			compression = "none";
 			offset = <0x00020000>;
 			hash {
@@ -46,7 +46,7 @@
 
 		btl-s4 {
 			description = "Habana BTL prstn 2";
-			data = /incbin/("s4.bin.le");
+			data = /incbin/("@FILES_PATH_1@s4.bin.le");
 			compression = "none";
 			offset = <0x00030000>;
 			hash {
@@ -56,7 +56,7 @@
 
 		btl-s5 {
 			description = "Habana BTL pcie 1";
-			data = /incbin/("s5.bin.le");
+			data = /incbin/("@FILES_PATH_1@s5.bin.le");
 			compression = "none";
 			offset = <0x00040000>;
 			hash {
@@ -66,7 +66,7 @@
 
 		btl-s6 {
 			description = "Habana BTL pcie 2";
-			data = /incbin/("s6.bin.le");
+			data = /incbin/("@FILES_PATH_1@s6.bin.le");
 			compression = "none";
 			offset = <0x00050000>;
 			hash {
@@ -76,7 +76,7 @@
 
 		img-ptr {
 			description = "Habana image pointers";
-			data = /incbin/("@HL_DEV@-images-pointers.bin.le");
+			data = /incbin/("@FILES_PATH_1@@HL_DEV@-images-pointers.bin.le");
 			compression = "none";
 			offset = <0x00060000>;
 			hash {
@@ -86,7 +86,7 @@
 
 		preboot {
 			description = "Habana Preboot";
-			data = /incbin/("@HL_DEV@-preboot.bin.le");
+			data = /incbin/("@FILES_PATH_1@@HL_DEV@-preboot.bin.le");
 			type = "standalone";
 			arch = "arm64";
 			compression = "none";
diff --git a/board/habanalabs/gaudi3/fake_fuse.its b/board/habanalabs/gaudi3/fake_fuse.its
index 2cd7a96..42fdc2a 100644
--- a/board/habanalabs/gaudi3/fake_fuse.its
+++ b/board/habanalabs/gaudi3/fake_fuse.its
@@ -11,7 +11,7 @@
 	images {
 		fake-fuse-image {
 			description = "Habana fake-fuse";
-			data = /incbin/("@DEV_PREFIX@-fake-fuse.bin.be");
+			data = /incbin/("@FILES_PATH_2@@DEV_PREFIX@-fake-fuse.bin.be");
 			compression = "none";
 			arch = "arc";
 			/* offset for fake fuse sector */
